Formula/ksync.rb

```diff
@@ -28,7 +28,7 @@ def install
       -w
       -X #{project}/pkg/ksync.GitCommit=#{Utils.git_short_head}
       -X #{project}/pkg/ksync.GitTag=#{version}
-      -X #{project}/pkg/ksync.BuildDate=#{Utils.safe_popen_read("date", "+%Y%m%dT%H%M%S").chomp}
+      -X #{project}/pkg/ksync.BuildDate=#{Time.now.utc.strftime("%Y%m%dT%H%M%S")}
```

Comment:
In this case, since `date` didn't use `-u`, so it wasn't using UTC time originally. Should it be switched or kept as-is?

Looks like upstream [does use UTC](https://github.com/ksync/ksync/blob/0fe3c20e49a7cbbba933951b56d3381cad083cc4/Makefile#L6) (but also uses RFC 3339 which doesn't match here):

```Makefile
BUILD_DATE      := $(shell date --utc --rfc-3339 ns 2> /dev/null | sed -e 's/ /T/')
```

On a Raspberry Pi I have lying around:

```console
$ date --utc --rfc-3339 ns 2> /dev/null | sed -e 's/ /T/'
2021-03-08T03:01:31.874011679+00:00
```

Reply:
Do whatever upstream does is, in general, my suggestion.

e.g. `DateTime.now.utc.rfc3339(9)`

Actually ends up to be simpler code.